stream-sdk/stream-starter-parents/stream-task-starter-parent/pom.xml

```diff
@@ -5,13 +5,13 @@
     <parent>
         <groupId>com.backbase.buildingblocks</groupId>
         <artifactId>service-sdk-starter-reactive</artifactId>
-        <version>15.2.0</version>
+        <version>15.2.1-cr.3-SNAPSHOT</version>
```

Comment:
Waiting for SSDK 15.2.1 release

Reply:
cool, let's just make sure we don't merge this since it would be relying on an internal release which is not available to our GitHub workflows currently.